content/en/releases/version-skew-policy.md

```diff
@@ -64,6 +64,7 @@ Example:
 ### kube-proxy
 
 `kube-proxy` must not be newer than `kube-apiserver`, and may be up to two minor versions older.
+`kube-proxy` may be up to two minor versions older or newer than the `kubelet` instance it runs alongside.
```

Comment:
kube-proxy being allowed to be *older* than kubelet was not necessarily true by implication, since kubelets cannot be newer than the control-plane. If we allow that, this needs to be stated explicitly.

Reply:
https://github.com/kubernetes/kubernetes/issues/117756 opened for ^